src/stories/components/DataCard/DataCard.js

```diff
@@ -0,0 +1,61 @@
+import './styles.scss';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Component for data card element.
+ *
+ * @component
+ * @param {string} modifierClasses of the data card
+ * @param {sting} title the data card
+ * @param {string} description of the data card
+ * @param {node} children of the data card
+ * @return {object}
+ *   <DataCard
+ *      modifierClasses={modifierClasses}
+ *      title={title}
+ *      description={description}
+ *      children={children} />
+ * )
+ */
+export const DataCard = (
+    { modifierClasses, title, description, children },
+) => {
+    return (
+        <div className={['data-card', `${modifierClasses}`].join(' ').trim()}>
+            <div className="data-card__container">
+                <div className="data-card__details">
+                    <p className="data-card__title">{title}</p>
```

Comment:
@denglish7 Update this to `<h6>` and you won't need to apply special styles. I realized there's an error in the global styles, sorry! Can you replace this [line](https://github.com/neu-self-image-experiments/sie-firebase/blob/4ab821ae0073ad1f40bf3170842420fbdb49e9f2/src/scss/_base.scss#L23) with `font-family: $header-font-family;`. Thank you!